gammapy/spectrum/tests/test_core.py

```diff
@@ -134,3 +137,12 @@ def test_rebin(self):
         assert (spec_rebinned.quality == [1, 0, 0, 1]).all()
         assert_quantity_allclose(spec_rebinned.hi_threshold, 5.623413251903491 * u.TeV)
         assert_quantity_allclose(spec_rebinned.lo_threshold, 1.778279410038922 * u.TeV)
+
+    @requires_dependency("sherpa")
+    def test_to_sherpa(self):
+        sherpa_dataset = self.spec.to_sherpa("test")
+        assert_allclose(sherpa_dataset.counts, self.spec.data.data)
+
+    def test_reset_thresholds(self):
+        self.spec.reset_thresholds()
+        assert_allclose(self.spec.quality, np.zeros(8))
```

Comment:
Replace `np.zeros(8)` -> `0` (the `assert_allclose` function or almost any Numpy function will broadcast inputs.

Reply:
Done